src/manage_sql.c

```diff
@@ -23816,6 +23875,82 @@ where_qod (int min_qod)
       "task_id",                                                              \
       KEYWORD_TYPE_STRING },                                                  \
     { "(SELECT cve FROM nvts WHERE oid = nvt)", "cve", KEYWORD_TYPE_STRING }, \
+    { "(SELECT value"                                                         \
+      " FROM report_host_details"                                             \
+      " WHERE report_host = (SELECT id"                                       \
+      "                      FROM report_hosts"                               \
+      "                      WHERE report = results.report"                   \
+      "                      AND host = results.host)"                        \
+      " AND name = 'detected_by'"                                             \
+      " AND source_name = results.nvt)",                                      \
```

Comment:
This also needs to be limited as above.

Reply:
Thanks, does 80515ff solve it?

Reply:
Yes, I no longer get the error with this.